src/packages/@common/types/workspace/request.ts

```diff
@@ -48,6 +48,7 @@ export enum RequestSectionEnum {
   AUTHORIZATION = "Authorization",
   HEADERS = "Headers",
   REQUEST_BODY = "Request Body",
+  DOCUMENTATION = "Description",
 }
```

Comment:
i think here it will be documentation

Reply:
Description is a predefined , defined in previous code .

Reply:
okay then make it consistent to documentation only

Reply:
`DOCUMENTATION = "Documentation"`